releases/v2.8.0.toml

```
@@ -0,0 +1,55 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "registry"
+github_repo = "distribution/distribution"
+
+# previous release
+previous = "v2.7.1"
+
+pre_release = false
+
+preface = """\
+The 2.8.0 registry release has been a long time overdue.
+This is the last 2.x release. No further active development will continue on
+2.x branch. Security vulnerability patches to 2.x might be considered, but
+all active development will be focussed on v3 release due next year.
```

```suggestion
all active development will be focussed on v3 release due in 2022.
```

Let's use absolute dates to keep this document evergreen